### ISSUE TITLE
ProjectRed bundled cables not working

### ISSUE THREAD

**Issue by raizosec** [MightyPirates/OpenComputers#2401]
The projectred bundled cables dont connect to the computer. i have installed a tier2 rs card. visually the dont seem to connect to the computer (in comparsion to normal covered wires).
OC version: 1.6.2.7
Projec Red: 4.8.1.30

**Comment by xarses** [comment]
Which mc version is this?

**Comment by Techokami** [comment]
From the mod versions, I'm going to assume 1.10.2, since P:R support does seem to be missing in that MC version...

**Comment by Vexatos** [comment]
But... [I did add it](https://github.com/MightyPirates/OpenComputers/commit/130084d931750f24ef6b1e247b43df66de597274).

**Comment by cartmen180** [comment]
Yet it is not working. Version 1.6.2.7 on the FTB Beyond pack (v1.9) for MC 1.10.2

**Comment by Spacemike** [comment]
I can confirm.
Minecraft Version: 1.10.2
Open Computers Version: 1.6.2.7
Project Red Version: 4.8.4.49
Forge Multipart Version: 2.0.0.38
Core Chicken Lib Version: 2.5.9.283
MrTJPCore Version: 2.0.1.20

I've also tried going back to Project Red Version 4.8.0.17 as I saw another post about this topic awhile ago, however, it seems that no matter what I try to do, the Project Red Bundled Cable refuses to connect to the computer case or Redstone I/O block, despite having a Tier 2 Redstone card in the computer case, and the cable seems to just ignore any input from the computer and vice versa, however, regular insulted cables work just fine, so I believe that either the bundled cable or the computer case is not seeing the other as a viable connection for some reason, maybe it has something to do with one of the updates?

**Comment by artheus** [comment]
I too can confirm this.
```
Minecraft Version:       1.10.2
Open Computers Version:  1.6.2.7
Project Red Version:     4.8.4.49
```

The bundled cable refuses to connect to the computer and/or the redstone i/o block.
Only thing I get working is the Red alloy wires and the insulated wires. But those will only work with the vanilla type of redstone.

**Comment by Poccket** [comment]
I think Project: Red has compatibility issues in 1.10.\*, as it doesn't work with any versions of ComputerCraft either, though ComputerCraft doesn't have a full release in 1.10.\*

**Comment by amesgen** [comment]
I can't reproduce this with the following versions:

 - Forge 12.18.3.2511 (Minecraft 1.10.2)
 - OpenComputers 1.7.2.170
 - ProjectRed 4.8.4.49
    - MrTJPCore 2.0.1.20
    - Forge Multipart 2.0.0.38
    - CodeChickenLib 2.5.9.320

Maybe it is irritating that the redstone strength is 0 - 255, and not 0 - 15. This could be noted in the documentation.

**Comment by artheus** [comment]
I found the reason for the crash. The reason for me was that I was using Dynmap to create a http map on my server. This made it look as though the cables were not working. But the reason was that Dynmap could not interpret the cable blocks. I removed Dynmap and everything worked!
So If anyone of you are using Dynmap, I suggest that you remove it. Instead, I think JourneyMap has a configuration which will allow you to see the map remotely in a similar fashion.
